tests/cxx/indirect.h

```diff
@@ -23,5 +23,16 @@ class IndirectClass {
   static int statica;
 };
 
+template <typename T>
+class IndirectTemplate {
+ public:
+  void Method() const {};
```

Comment:
No semicolon after inline method body, here or below.

And same thing here, if you want to clean up `IndirectClass` by removing them there too, please do so in a separate commit.